### DISCUSSION TITLE
how to detect the pasted image?

### DISCUSSION THREAD

**Discussion by SunnyCheung-cp** [Milkdown/milkdown#1159] [Q&A]
I'm using `@milkdown/plugin-upload`
When I copy the base64 image and paste it, it uploads successfully
But when I copied the link of the image, it was not detected

What do I need to do to detect the pasted image?

https://github.com/Milkdown/milkdown/assets/105030733/3d3ade32-93d6-405e-be2f-5f8899a59562

**Comment by Saul-Mirone** [comment]
You'll need to configure [this option](https://milkdown.dev/docs/api/plugin-upload#interface-upload-options).

The `enableHtmlFileUploader` config is `false` by default.
